tests_integration/test_streamlit.py

```diff
@@ -367,3 +368,50 @@ def _new_streamlit_role(snowflake_session, test_database):
         row_from_snowflake_session(result),
         {"status": f"{role_name.upper()} successfully dropped."},
     )
+
+
+def _execute_streamlit(runner, streamlit_name):
+    result = runner.invoke_with_connection_json(
+        ["streamlit", "execute", streamlit_name, "--format", "json"]
+    )
+    assert result.exit_code == 0
+    assert result.json == {"message": f"Streamlit {streamlit_name} executed."}
+
+
+def _execute_streamlit_failure(runner, streamlit_name):
+    result = runner.invoke_with_connection(["streamlit", "execute", streamlit_name])
+    assert result.exit_code == 1
+    assert f"NameError: name {streamlit_name} is not defined" in result.output
```

Comment:
Do we use any of those functions?

Reply:
I forgot to remove them, in first try I was using them as a helper function and then decide to try to execute directly inside function which is worked and forgot to remove them.